plugins/mac-vth264/encoder.c

```diff
@@ -904,6 +1006,16 @@ static obs_properties_t *vt_h264_properties(void *unused)
 
 static void vt_h264_defaults(obs_data_t *settings)
 {
+#if defined(__aarch64__)
+	if (__builtin_available(macOS 13.0, *)) {
+		obs_data_set_default_int(settings, "rate_control",
+					 H264RateControlCBR);
+	} else
+#endif
+	{
+		obs_data_set_default_int(settings, "rate_control",
+					 H264RateControlABR);
+	}
```

Comment:
Because of how we store settings, we can't just change defaults, it would instead need to have a v2 source.
We plan to push this PR to a separate branch anyways so that we can also incorporate two separate existing PRs, so if you're unfamiliar with this, we can probably do that ourselves if you want us to.

Reply:
i don't see how changing the module name could fix this - the stored encoder ID will still (correctly) contain h264

Reply:
As discussed off-thread we are fine without fixing this for now.